.circleci/config.yml

```diff
@@ -17,20 +17,13 @@ jobs:
     steps:
     - prometheus/setup_environment
     - run:
-        command: make style unused vet build
+        command: make all
         environment:
           # By default Go uses GOMAXPROCS but a Circle CI executor has many
           # cores (> 30) while the CPU and RAM resources are throttled. If we
           # don't limit this to the number of allocated cores, the job is
           # likely to get OOMed and killed.
           GOOPTS: "-p 2"
-    - run: git diff --exit-code
-    - prometheus/store_artifact:
```

Comment:
the same with these, why do we need them now?

Reply:
don't think we need them here, in prometheus these are used to build the docker images afaik.

Reply:
yes this is what I thought.